po/vi.po

```diff
@@ -6128,58 +6128,68 @@ msgstr ""
 
 #: src/std-types/string.md
 msgid "String"
-msgstr ""
+msgstr "String"
 
 #: src/std-types/string.md
 msgid ""
 "[`String`](https://doc.rust-lang.org/std/string/struct.String.html) is the "
 "standard heap-allocated growable UTF-8 string buffer:"
 msgstr ""
+"[`String`](https://doc.rust-lang.org/std/string/struct.String.html) là kiểu "
```

Comment:
String can be translated to Xâu ký tự. If you are unsure about the term translation, you can translate using the following format "English Term (Bản dịch tiếng Việt)" and proceed to go on to the part that can be easily translated. 

So it should be the following the format:
"[`String (Xâu ký tự)`] là kiểu (https://doc.rust-lang.org/std/string/struct.String.html) là kiểu " 
"dữ liệu chuẩn sử dụng để lưu trữ chuỗi UTF-8 trên bộ nhớ đệm xâu ký tự (buffer string) sử dụng cấu trúc chất đống (heap), có thể được mở rộng:"